### PR TITLE
[ci] Get rid of new Ruby 1.9 hash syntax messages

### PR DESCRIPTION
**Pull request by eduardoj**
Get rid of 'RuboCop: Use the new Ruby 1.9 hash syntax.' warnings in
Haml-lint.

### CODE REVIEW

**Comment by eduardoj** [comment]
[Enable Rubocop HAML-Lint linter](https://trello.com/c/RYPehyM5/359-enable-rubocop-haml-lint-linter)

File: src/api/app/views/webui/theme/bratwurst/layouts/webui/_navbar.html.haml [hunk 1]
```diff
@@ -1,14 +1,14 @@
-.navbar.navbar-default.navbar-fixed-top{:role => "navigation"}
+.navbar.navbar-default.navbar-fixed-top{role: "navigation"}
   / Brand and toggle get grouped for better mobile display
   .container
     .navbar-header
-      %button.navbar-toggle{"data-target" => ".navbar-ex1-collapse", "data-toggle" => "collapse", :type => "button"}
+      %button.navbar-toggle{"data-target" => ".navbar-ex1-collapse", "data-toggle" => "collapse", type: "button"}
```
Comment (DavidKang) [inline src/api/app/views/webui/theme/bratwurst/layouts/webui/_navbar.html.haml]: This could be something like:
```
{ data: { target: '.navbar-ex1-collapse', toggle: 'collapse' } }

File: src/api/app/views/webui/theme/bratwurst/layouts/webui/_alert.html.haml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 - flash.each do |type, message|
-  %div{:class=>"alert alert-dismissable #{bootstrap_class_for(type)}", :id=>"flash"}
-    .button.close{"data-dismiss" => "alert", "aria-hidden"=>"true"}
+  %div{class: "alert alert-dismissable #{bootstrap_class_for(type)}", id: "flash"}
+    .button.close{"data-dismiss" => "alert", "aria-hidden" => "true"}
```
Comment (DavidKang) [inline src/api/app/views/webui/theme/bratwurst/layouts/webui/_alert.html.haml]: This could be like:
```
{ data: { dismiss: 'alert' },  aria: { hidden: 'collapse' }  }

File: src/api/app/views/webui/theme/bratwurst/layouts/webui/_navbar.html.haml [hunk 3]
```diff
@@ -21,7 +21,7 @@
       - else
         %ul.nav.navbar-nav.navbar-right.hidden-xs
           %li.dropdown
-            %a.dropdown-toggle{"data-toggle" => "dropdown", :href => "#", :id => "user-dropdown"}
+            %a.dropdown-toggle{"data-toggle" => "dropdown", href: "#", id: "user-dropdown"}
```
Comment (DavidKang) [inline src/api/app/views/webui/theme/bratwurst/layouts/webui/_navbar.html.haml]: Same here

File: src/api/app/views/webui/theme/bratwurst/layouts/webui/_navbar.html.haml (hunk 1 above)
Comment (eduardoj) [inline src/api/app/views/webui/theme/bratwurst/layouts/webui/_navbar.html.haml]: Wouldn't that change you propose be part of another type of fix than "Get rid of 'RuboCop: Use the new Ruby 1.9 hash syntax.' warnings in Haml-lint."?

Can we agree in accept this pull request, and add your change in another one?

File: src/api/app/views/webui/theme/bratwurst/layouts/webui/_navbar.html.haml (hunk 3 above)
Comment (eduardoj) [inline src/api/app/views/webui/theme/bratwurst/layouts/webui/_navbar.html.haml]: Same response

File: src/api/app/views/webui/theme/bratwurst/layouts/webui/_alert.html.haml (hunk 2 above)
Comment (eduardoj) [inline src/api/app/views/webui/theme/bratwurst/layouts/webui/_alert.html.haml]: Same response

File: src/api/app/views/webui/theme/bratwurst/layouts/webui/_navbar.html.haml (hunk 1 above)
Comment (DavidKang) [inline src/api/app/views/webui/theme/bratwurst/layouts/webui/_navbar.html.haml]: We can do it in another PR
